src/porepy/utils/comp_geom.py

```diff
@@ -1450,15 +1456,56 @@ def is_planar(pts, normal=None):
     if normal is None:
         normal = compute_normal(pts)
     else:
-        normal = normal / np.linalg.norm(normal)
+        normal = normal.flatten() / np.linalg.norm(normal)
+
+    check_all = np.zeros(pts.shape[1]-1, dtype=np.bool)
+
+    for idx, p in enumerate(pts[:, 1:].T):
+        den = np.linalg.norm(pts[:, 0] - p)
+        dotprod = np.dot(normal, (pts[:, 0] - p)/(den if den else 1))
+        check_all[idx] = np.isclose(dotprod, 0, atol=tol, rtol=0)
+
+    return np.all(check_all)
+
+#------------------------------------------------------------------------------#
+
+def is_point_in_cell(pts, pt, if_make_planar=True):
```

Comment:
duplicate of cg.is_inside_polygon()? With the exception of the rotate option.

Reply:
the new algorithm works also for concave cells, what should I do? keep both?